GameData/RP-0/Science/CrewScience/CrewLabAdder.cfg

```diff
@@ -68,7 +68,7 @@ RESOURCE_DEFINITION
 //**********************************************************************************
 // 	SECOND GEN CAPSULES
 //**********************************************************************************
-@PART[FASAGeminiPod2|FASAGeminiPod2White|K2Pod|Voskhod_Crew_A]:FOR[RP-0]
+@PART[FASAGeminiPod2|FASAGeminiPod2White|K2Pod|Voskhod_Crew_A|RO-Mk1CrewModule|RO-Mk1CockpitInline]:FOR[RP-0]
```

Comment:
Any reason for no `RO-Mk1Cockpit` here?

Reply:
Probably just an oversight. Or maybe, isn't that the X-15 cockpit? Possibly I just honored the new rule that the X-15 may not under any circumstances be suitable for spaceflight.

Reply:
No, the X-15 is `Mark1Cockpit`, which is a one-seater.  `RO-Mk1Cockpit` and `RO-Mk1CockpitInline` are two-seater cockpits in tech Prototype Spaceplaces[sic].  Thus if the `Inline` version is here (as a two-seat space-capable cockpit) so should the non-`Inline` one.

Reply:
Added it